index.html

```diff
@@ -1,241 +1,366 @@
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
           </p>
-          <ul class="works-item-list">
-            <li>HTML</li>
-            <li>Bootscrap</li>
-            <li>Ruby</li>
-          </ul>
+          <button type="button" class="about-btn btn">Get my resume</button>
+        </article>
+        <hr />
+        <article class="about-list">
+          <div class="language">
+            <h3 class="heading">Language</h3>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+          <div class="framework">
+            <h3 class="heading">Frameworks</h3>
+            <ul>
+              <li>React.js</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+            </ul>
+          </div>
+          <div class="skills">
+            <h3 class="heading">Skills</h3>
+            <ul>
+              <li>Data Management</li>
+              <li>Version Control</li>
+              <li>CLI</li>
+              <li>API Design</li>
+              <li>Web Development</li>
+            </ul>
+          </div>
         </article>
-        <button type="button" class="works-btn btn">See project</button>
       </section>
-
-    </section>
-
-
-    <section class="about-section">
-      <article class="about-details">
-        <h3>About me</h3>
-        <p>Hello I’m a software developer! I can help you build a product,
-          feature or website Look through some of my work and experience!
-          If you like what you see and have a project you need coded, don’t
-          hesitate to contact me.
+      <section class="contact-section">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
         </p>
-        <button type="button" class="about-btn btn">Get my resume</button>
-      </article>
-      <hr>
-      <article class="about-list">
-        <div class="language">
-          <h3 class="heading">Language</h3>
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-        </div>
-        <div class="framework">
-          <h3 class="heading">Frameworks</h3>
-          <ul>
-            <li>React.js</li>
-            <li>Ruby on Rails</li>
-            <li>RSpec</li>
-            <li>Capybara</li>
-          </ul>
-        </div>
-        <div class="skills">
-          <h3 class="heading">Skills</h3>
-          <ul>
-            <li>Data Management</li>
-            <li>Version Control</li>
-            <li>CLI</li>
-            <li>API Design</li>
-            <li>Web Development</li>
-          </ul>
-        </div>
-      </article>
-    </section>
-  </main>
-</body>
-
-</html>
\ No newline at end of file
+        <form
```

Comment:
Great work on implementing the contact and the footer, but there are some changes you'll need to perform to go to the next step.
Kindly, I suggest you stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding) to make your project look more professional.

> ## The Figma Design
>
><img width="337" alt="Screenshot 2023-06-06 at 12 31 34 PM" src="https://github.com/Wineshuga/portfolio/assets/91237525/0987c894-e3ab-4e05-bacf-2c7f24b4d60e">
>
> ## Your design
>
><img width="376" alt="Screenshot 2023-06-06 at 12 33 52 PM" src="https://github.com/Wineshuga/portfolio/assets/91237525/4135d08c-b397-465c-9723-78ba978069e5">
><img width="374" alt="Screenshot 2023-06-06 at 12 34 02 PM" src="https://github.com/Wineshuga/portfolio/assets/91237525/96cd3eeb-c005-47fc-bf9c-e564c7addc60">